src/lib/util/Config.ts

```diff
@@ -79,6 +90,31 @@ export class Config {
 		return this.configContent.engines.find(e => e.name === name);
 	}
 
+	public getOverriddenConfigPath(name: string): string {
+		const defaultValue = '';
+		const engineConfig = this.getEngineConfig(name);
+
+		if (!this.shouldUseDefaultConfig(name)) {
+			if (!engineConfig || !engineConfig.overriddenConfigPath) {
```

Comment:
Style question, in other places the code uses 'if (engineConfig == null ||'. Which is the team's style?

Reply:
I guess we don't have an established style :)

Reply:
Planning to stick with (!engineConfig) for this PR.